web/pages/[workspaceSlug]/projects/[projectId]/pages/[pageId].tsx

```diff
@@ -39,37 +38,23 @@ const PageDetailsPage: NextPageWithLayout = observer(() => {
   // store hooks
   const { createPage, getPageById } = useProjectPages(projectId?.toString() ?? "");
   const pageStore = usePage(pageId?.toString() ?? "");
```

Comment:
Can you rename this to page. This is an instance of page.

Reply:
Renamed `pageStore` to `page`.